bazel/api_build_system.bzl

```diff
@@ -77,7 +77,7 @@ def api_go_grpc_library(name, proto, deps = []):
 # gRPC stub generation.
 # TODO(htuch): Automatically generate go_proto_library and go_grpc_library
 # from api_proto_library.
-def api_proto_library(name, srcs = [], deps = [], has_services = 0, require_py = 1):
+def api_proto_library(name, visibility, srcs = [], deps = [], has_services = 0, require_py = 1):
```

Comment:
Should this default to `"//visibility:public"` for backward-compatibility?

Reply:
No strong opinion. I found it useful for myself to make sure I didn't miss any.